build.sbt

```diff
@@ -25,20 +31,54 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = Some("2.6.0-M1") // first from this separate repo
```

Comment:
That's what I wanted, but due to the package name change, it will detect everything as changed